VAMobile/src/api/notifications/getSystemNotificationSettings.tsx

```diff
@@ -0,0 +1,30 @@
+import { useQuery } from '@tanstack/react-query'
+
+import { LoadSystemNotificationsData } from 'api/types'
+import { notificationsEnabled } from 'utils/notifications'
+
+import { notificationKeys } from './queryKeys'
+
+/**
+ * Fetch user system notification settings
+ */
+const getSystemNotificationsSettings = async (): Promise<LoadSystemNotificationsData | undefined> => {
+  const systemNotificationsOn = await notificationsEnabled()
+  return {
+    systemNotificationsOn,
+  }
+}
+
+/**
+ * Returns a query for user system notification settings
+ */
+export const useSystemNotificationsSettings = (options?: { enabled?: boolean }) => {
+  return useQuery({
+    ...options,
+    queryKey: notificationKeys.systemSettings,
+    queryFn: () => getSystemNotificationsSettings(),
+    meta: {
+      errorName: 'loadPushPreferences: Service error',
```

Comment:
```suggestion
      errorName: 'getSystemNotificationsSettings: Failed to retrieve system notification setting',
```

Wrong function was named here, also the error message should be updated since this wouldn't be a service error

Reply:
updated